packages/tools/pie-css/scss/resets/index.scss

```diff
@@ -0,0 +1,21 @@
+html {
```

Comment:
You shouldn't need the SCSS part – box-sizing should be applied via the imported CSS ([defined here](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-css/css/input.css)), which consuming apps should already be pulling in.

So you should be able to just remove the box-sizing rules without adding to the SCSS.